.github/workflows/test-coderabbit-trigger.yml

```
@@ -0,0 +1,16 @@
+name: Test CodeRabbit Trigger
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'PR number to test with'
+        required: true
+        type: number
+
+jobs:
+  test:
+    uses: ./github/workflows/coderabbit-trigger.yml@add-coderabbit-dependabot-workflow
```

actionlint (1.7.9)

reusable workflow call "./github/workflows/coderabbit-trigger.yml@add-coderabbit-dependabot-workflow" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)